PayloadPkg/OsLoader/OsLoader.inf

```diff
@@ -40,6 +40,7 @@
 
 [Packages]
   MdePkg/MdePkg.dec
+  BootloaderCorePkg/BootloaderCorePkg.dec
```

Comment:
The code change looks good.
Remember we try to avoid to use BootloaderCorePkg in OsLoader. is it still the case now?

Reply:
Not sure, @mauricema any thoughts on that before we merge?

Reply:
I agree with @gdong1. Accessing BootloaderCorePkg from Payload is not a good option.

Reply:
In general, it is true for payload.  But for OsLoader payload, it is considered as special since it is always built together with SBL core code.  
In this case, it is better if we can avoid any reference to BootloaderCorePkg.dec.  I just looked into the code,  since we passed PldBase to OsLoader, can we use this to do the comparison instead of the PCDs ?

Reply:
Never mind.  Noticed that PldBase is only used for PreOsChecker.
I think maybe we can use the symbol  ModuleEntryPoint to get the OsLoader loading base.